docs/snippets/quickstart-install.md

```diff
@@ -118,3 +121,12 @@ To get a local deployment of Knative, run the `quickstart` plugin:
         ```bash
         minikube profile list
         ```
+    1. To finish setting up networking for minikube, start the `minikube tunnel` process in a separate terminal window:
```

Comment:
@snneji  only Windows/Mac users need to manually do this step (discussion of what that's the case is in https://github.com/knative-sandbox/kn-plugin-quickstart/pull/139 )

Reply:
@psschwei I'll add an issue to clarify this. 

If Linux users do this step, does it cause any issues?